### PR TITLE
Feature/restructure settings module

### LINKED ISSUE #1474: Restructure settings module
The settings module code does not follow the structure of other project modules. Re-structure the Settings module to follow the [project structure conventions](https://github.com/apinf/api-umbrella-dashboard/blob/develop/CONTRIBUTING.md#file-structure).

# original issue
The Settings form does not work in `develop`. This may be due to switching the collection to using `export` / `import` syntax. Find the underlying issue and fix it.

### PR DESCRIPTION
**Pull request by brylie**
Closes #1474 

# Changes
- Import Settings collection in settings file(s)
- Create template helper for settingsCollection, used by AutoForm
- Cleanup AutoForm hooks
- Remove unneeded Settings schema fields
- Automatic linting
- Remove unused API Umbrella methods

### CODE REVIEW

**Comment by bajiat** [comment]
@marla-singer Would you be available to review Brylie's PR? @brylie deleted the previous one you were reviewing, because it was giving console errors. This should not have the errors.

**Comment by marla-singer** [comment]
@bajiat Yes, i'm going to review it now

File: settings/client/autoform.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+AutoForm.addHooks(['settings'], {
+  onSuccess () {
+    FlashMessages.sendSuccess('Settings saved.');
```
Comment (marla-singer) [inline settings/client/autoform.js]: Add i18n token

**Comment by marla-singer** [comment]
@brylie It works good

File: settings/client/autoform.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+AutoForm.addHooks(['settings'], {
```
Comment (nnn) [inline settings/client/autoform.js]: Let's use AutoForm.hooks instead of AutoForm.addHooks.

File: settings/client/autoform.js (hunk 1 above)
Comment (nnn) [inline settings/client/autoform.js]: Let's switch to sAlert notifications only. Now we have both FlashMessages and sAlert.
For this PR is enough to change it to sAlert just here, but we need have a separate task to completely remove FlashMessages package from the project.

**Comment by brylie** [comment]
@marla-singer and @nnn I made the suggested changes. Please review APAP.

Ping @bajiat.

**Comment by marla-singer** [comment]
@brylie Everything is okay. I can merge or waiting for @nnn check?

**Pull request merged**